speedy-derive/src/lib.rs

```diff
@@ -531,25 +538,30 @@ fn is_c( attrs: &[syn::Attribute] ) -> bool {
 }
 
 fn parse_attributes< T >( attrs: &[syn::Attribute] ) -> Result< Vec< T >, syn::Error > where T: syn::parse::Parse {
-    struct RawAttributes< T >( syn::punctuated::Punctuated< T, Token![,] > );
+    struct RawAttributes< T >( Punctuated< T, Token![,] > );
 
     impl< T > syn::parse::Parse for RawAttributes< T > where T: syn::parse::Parse {
         fn parse( input: syn::parse::ParseStream ) -> syn::parse::Result< Self > {
+            dbg!(&input);
```

Comment:
@koute Sorry but I committed this by accident 😕 Should probably remove it before releasing.

Reply:
Yeah, don't worry about it; I've already noticed it and removed it.